Site/ASAB Maestro/Descriptors/kibana.yaml

```diff
@@ -0,0 +1,90 @@
+define:
+  type: rc/descriptor
+  name: Kibana
+  url: https://www.elastic.co/kibana
+
+descriptor:
+  image: docker.elastic.co/kibana/kibana
+  environment:
+    SERVER_HOST: "{{NODE_ID}}"
+  volumes:
+    - "{{SITE}}/{{INSTANCE_ID}}/config:/usr/share/kibana/config"
+
+
+files:
+  - "config/kibana.yml": |
+      elasticsearch.hosts: {{ELASTIC_HOSTS_KIBANA}}
+      elasticsearch.username: "elastic"
+      elasticsearch.password: "{{ELASTIC_PASSWORD}}"
+      xpack.monitoring.ui.container.elasticsearch.enabled: true
+      server.publicBaseUrl: "{{PUBLIC_URL}}/kibana"
+      server.basePath: "/kibana"
+      server.rewriteBasePath: true
+
+# server.publicBaseUrl: "{{PUBLIC_URL}}/kibana" -> kibana is directly dependent on nginx tech - on nginx present in the model
+# nginx locations are dependent on seacat auth presence
+
+seacat-auth:
+  config:
+    "batman:elk":
+      url: "http://{{NODE_ID}}:9200" # <- this is not general enough
```

Comment:
Yes, this has to point to 1 or more ES nodes in the cluster.
It could even result to a enhancement in `batman:elk` to support this.

Ie. to accept `{{ELASTIC_HOSTS_KIBANA}}`

Reply:
it accepts just one url right now. refactoring in seacat auth is needed.

Reply:
I know.

Reply:
https://github.com/TeskaLabs/seacat-auth/issues/302#issuecomment-1759876263